docs/zh/UserGuide/Operators-Functions/Aggregation.md

```diff
@@ -104,4 +105,53 @@ select count_if(s1=0 & s2=0, 3, 'ignoreNull'='false'), count_if(s1=1 & s2=0, 3,
 +------------------------------------------------------------------------+------------------------------------------------------------------------+
 |                                                                       1|                                                                       1|
 +------------------------------------------------------------------------+------------------------------------------------------------------------+
+```
+
+## TIME_DURATION
+### 语法
+```sql
+select
+    time_duration(Path),[COMMA time_duration(Path)]...
```

Comment:
we just need `time_duration(Path)` & keep doc same between English and Chinese doc.